qemu/tests/virtio_console.py

```diff
@@ -280,7 +280,7 @@ def test_rw_host_offline_big_data():
         guest_worker.cmd("virt.clean_port('%s'),1024" % port.name, 10)
         match, tmp = guest_worker._cmd("virt.send('%s', (1024**3)*3, True, "
                                        "is_static=True)" % port.name, 30)
-        if match is None:
+        if match is not None:
```

Comment:
@sitoliu 
from qemu_virtio_port.py file.  If match is not None, It should be 0 or 1?
def _cmd(self, cmd, timeout=10, patterns=None):
..........
        try:
            (match, data) = self.session.read_until_any_line_matches(patterns,
                                                                     timeout=timeout)
            if patterns[match].startswith('^PASS:'):
                match = 0
            elif patterns[match].startswith('^FAIL:'):
                match = 1
            else:
                data = ("Incorrect pattern %s. Data in console:\n%s"
                        % (patterns[match], data))
                match = None